demos/continuous_batching/README.md

```diff
@@ -7,12 +7,7 @@ That makes it easy to use and efficient especially on on Intel® Xeon® processo
 
 ## Get the docker image
 
-Pull public image with CPU only support or including also GPU support.
-```bash
-docker pull openvino/model_server:latest-gpu
-docker pull openvino/model_server:latest
-```
-or build the image from source to try the latest enhancements in this feature.
+Build the image from source to try the latest enhancements in this feature.
```

Comment:
We no longer want to inform users that they can pull image from dockerhub?
Do we really want to replace this phrase on release branch every release as release procedure step? If so, please add note to wiki so we remember about this step every month